scripts/staging_start.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+service rustodon start
```

Comment:
Where is the `rustodon` service defined?  I don't see anything in the PR that looks like it would provide this and I got curious.

Is this handled e.g. by AWS CodeDeploy via `appspec.yml`?

Reply:
Yeah, right now the codedeployer agent calls that script based on its setting in appspec.yml

I wrote a service script for rustodon but it's pretty specific to that particular server at the moment. I know in many cases those scripts get left to the distribution package maintainer so I wasn't sure if they should be in the main codebase.

(The configuration of this CD server is pretty lightweight and manual right now, I don't have a full story yet around how to easily reproduce it. It would be nice if that was also reproducible and stored in git via some config manager but we're not there yet. And I imagine it would also generally not live in this codebase. I tried to find github examples where the appspec.yml scripts weren't in the main codeline but couldn't find any so I am assuming this is the common paradigm.)

Reply:
I'd definitely like it if we could, at the very minimum, bundle a generic service script that anyone could use (even if they weren't doing CD, just deploying rustodon). Is that possible?

Reply:
I can for now bundle one that puts our customizations as variables in the script, which people could modify. I have one for sysvinit (I don't yet have a systemd place to test a unit file, but I'd like to set that up in the future.)

It probably won't be out-of-the-box because based on my naive understanding of Rust packaging. But it at least would be an example.

Reply:
I added in the /etc/init.d file I'm currently using.